### PR TITLE
Made prefixes per-tool customizable

### PR DESCRIPTION
**Pull request by whitead**
Now can do things like prefix gmx_mpi:mdrun and gmx:grompp and the commands will execute using the different prefixes (called drivers in code). Addresses Issue #48

### CODE REVIEW

**Comment by dotsdl** [comment]
Thanks @whitead! I've assigned this to @orbeckst because he knows better the technical implications, but it looks good to me. You can ignore the Travis failure; we're still working to get CI up and running here.

**Comment by whitead** [comment]
@dotsdl Thank you! Hold off on merging though. I realized that the multi-index commands aren't going to work with this. I'm working on it.

**Comment by whitead** [comment]
Ok, fixed those issues. Should work now

File: gromacs/core.py [hunk 1]
```diff
@@ -594,7 +600,8 @@ def _get_gmx_docs(self):
 
         .. Note::
 
-           The header is on STDOUT and is ignored. The docs are read from STDERR.
+           The header is on STDOUT and is ignored. The docs are read from STDERR in GMX 4. 
+           In GMX 5, the opposite is true (Grrr)
```
Comment (orbeckst) [inline gromacs/core.py]: Oh, that's what it was... argh. Good catch!

**Comment by orbeckst** [comment]
Looking good! 

Did you manually test with Gromacs 5.x and also a 4.6.x? (Sorry, we *really* need the tests going... #23).

**Comment by whitead** [comment]
I tested 5.0, 5.1. I don't have a 4.6 handy right now.

**Comment by whitead** [comment]
Ok, managed to get 4.5.5 working. Testing it now.

**Comment by whitead** [comment]
Alright, can run a few commands on 4.5.5.

**Comment by whitead** [comment]
Hey, I forgot to change branches and pushed some extra crap. Let me reopen with correct commits.

**Comment by orbeckst** [comment]
Thanks. I’ll test 4.6.

> On 24 May, 2016, at 13:07, Andrew White <notifications@github.com> wrote:
> 
> I tested 5.0, 5.1. I don't have a 4.6 handy right now.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> 

--
Oliver Beckstein * orbeckst@gmx.net
skype: orbeckst  * orbeckst@gmail.com

**Pull request closed without merging**